src/components/button.js

```diff
@@ -20,7 +20,8 @@ export default function Button(props) {
           border: "1px solid " + primary,
           borderRadius: 20,
           backgroundColor: primary,
-          "@keyframes fadeInDivRev": {
```

Comment:
Why did you remove the hover animation of button? Is there any specific reason?
If no, then please revert it back and we are good to merge.